clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -30,25 +30,27 @@
 import org.junit.jupiter.api.DisplayName;
 import org.springframework.beans.factory.annotation.Autowired;
 
+/**
+ * TODO (allisaurus): Ideally these tests would go further and actually assert that the resulting
+ * ecs:create-service call is formed as expected, but for now, it asserts that the given operation
+ * is correctly validated and submitted as a task.
+ */
 public class CreateServerGroupSpec extends EcsSpec {
 
   @Autowired AccountCredentialsRepository accountCredentialsRepository;
 
+  String path = "/ecs/ops/createServerGroup";
```

Comment:
can we name and scope this var [as we do elsewhere](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/EcsSpec.java#L52) in this pkg? I.e.:
`private static final String CREATE_SG_TEST_PATH = ...`